content/news/preview-20190606.md

```diff
@@ -0,0 +1,59 @@
+It has been quite a while since our last release, but not to worry, we are still going - mostly rewriting tons of code not visible to the outside eye. While not presenting a full playtest we want to take the time to introduce the community to some upcoming changes to look forward to.
+
+### Singleplayer savegames
+
+Probably the most wanted missing feature, the next version will include support for savegames in both singleplayer and the campaign. No more losing all progress to a stray dog eating your Tanya!
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190606-savegame-d2k.png" alt="The savegame load screen in D2k." width="600px">
+
+Longer missions can now be played in several takes instead of needing you to pull through in one go.
+</div>
+
+### Overhauled UI
+
+Our spectator UI has always been noted to be clunky and taking too much space off the screen. With the next update we are happy to finally introduce a more slim and focused spectator UI.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190606-spectator-UI.png" alt="A new spectator UI." width="600px">
+
+The new transpart overlay is a huge quality of live upgrade for both professional casters and casual spectators.
+</div>
+
+Our second most visible update features yet another long standing UI rework that was just completed last month. The sidebar in Tiberian Dawn got completely overhauled and looks as shiny as ever.
+
+### Polished building placement previews
+
+There is not much to say here, the images will tell the story. The new placement preview not only shows a preview of the building but additionally makes it clearer which terrain is actually buildable.
```

Comment:
This suggests that we are adding the building preview for the first time. How about `The tweaked placement preview makes it easier to see both the building being placed and any cells that are blocked by units or debris`?